CHANGELOG.next.toml

```diff
@@ -11,6 +11,18 @@
 # meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client | server | all"}
 # author = "rcoh"
 
+[[smithy-rs]]
+message = "The `Debug` implementation for `PropertyBag` now prints a list of the types it contains. This significantly improves debuggability."
+author = "rcoh"
+references = ["smithy-rs#2612"]
+meta = { "breaking" = false, "tada" = false, "bug" = false }
+
+[[smithy-rs]]
+message = "Implement `Ord` and `PartialOrd` for `DateTime`."
+author = "henriiik"
+references = ["smithy-rs#2653", "smithy-rs#2656"]
+meta = { "breaking" = false, "tada" = false, "bug" = false }
+
```

Comment:
These were deleted in main:
```suggestion
```

Reply:
Yeah those got pulled in by accident with a rebase. whoops